### PR TITLE
Added new date component

### PR DESCRIPTION
**Pull request by michielglibert**

### CODE REVIEW

File: stories/input.stories.js [hunk 1]
```diff
@@ -566,6 +567,76 @@ storiesOf('Low level blocks/Inputs', module)
...
+      '',
+    );
+    const [returnedOnFieldBlurDate, setReturnedOnFieldBlurDate] = useState('');
+
+    return (
+      <Flex>
+        <StyledDateInputNew
+          name="dateInput"
+          disabled={boolean('Disabled', false)}
+          onChange={val => setReturnedOnChangeDate(val)}
```
Comment (amants) [inline stories/input.stories.js:583]: can just do 
onChange={setReturnedOnChangeDate}

File: stories/input.stories.js [hunk 2]
```diff
@@ -566,6 +567,76 @@ storiesOf('Low level blocks/Inputs', module)
...
+          onBlur={val => setReturnedOnBlurDate(val)}
```
Comment (amants) [inline stories/input.stories.js:584]: onBlur={setReturnedOnBlurDate}

File: stories/input.stories.js [hunk 3]
```diff
@@ -566,6 +567,76 @@ storiesOf('Low level blocks/Inputs', module)
...
+          onFieldChange={val => setReturnedOnFieldChangeDate(val)}
+          onFieldBlur={val => setReturnedOnFieldBlurDate(val)}
```
Comment (amants) [inline stories/input.stories.js:586]: onFieldChange={setReturnedOnFieldChangeDate}
onFieldBlur={setReturnedOnFieldBlurDate}

File: stories/input.stories.js [hunk 4]
```diff
@@ -587,6 +658,35 @@ storiesOf('Low level blocks/Inputs', module)
...
+`;
+
+const H2 = styled.h2`
+  margin-top: 0.5em;
+  font-size: 2em;
+`;
+
+const P = styled.p`
+  font-size: 1.5em;
+  margin: 0.5em 0;
```
Comment (amants) [inline stories/input.stories.js:673]: why the sudden change to em?

**Review by amants — CHANGES_REQUESTED** [review]

File: src/components/Inputs/DateInput_New.jsx [hunk 5]
```diff
@@ -0,0 +1,404 @@
+/* eslint-disable no-unused-vars */
```
[the remark this answers is not in the document]
Comment (michielglibert) [reply src/components/Inputs/DateInput_New.jsx:1]: XD

**Comment by amants** [comment]
Screenshot 2020-12-17 at 18 06 02
When leaving a certain dateinput field the data isn't padded

**Comment by amants** [comment]
when a input (day, month) is filled, the next input should be focused

**Comment by amants** [comment]
Screenshot 2020-12-17 at 18 08 23
Iso dates don't match the inputted date

**Comment by amants** [comment]
When in the month field a number 2 or higher is inputted, should default fill with 02 and go to next field.
When day field a number 4 or higher is inputted, should default fill with 04 and go to next field

File: stories/input.stories.js (hunk 4 above)
Comment (michielglibert) [reply stories/input.stories.js:674]: ow my bad, I normally use EM outside wegroup, wasn't thinking for a sec

File: src/components/Inputs/DateInputV4.jsx [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+      setReturnType('object');
+      return {
+        day: pad(value.day),
+        month: pad(value.month),
+        year: value.year,
+      };
+    } else if (isISODate(value)) {
+      setReturnType('iso');
+      const isoDate = new Date(value);
+      console.log(isoDate);
```
Comment (amants) [inline src/components/Inputs/DateInputV4.jsx:84]: rm log

**Pull request merged**